### ISSUE TITLE
Python interactive process restarts before every "Execute File in Python Interactive" run

### ISSUE THREAD

**Issue by CyanBC** [microsoft/PTVS#3400]
Python interactive window will restart before running a script under "Execute File in Python Interactive".

The restart is also associated with:
```
[IPKernalApp] WARNING | Parent appears to have exited, shutting down.
The interactive python process has exited.
```
before running the code.

Ctrl+Enter works fine without restarting python interactive, but is not convenient.

I do not know if this is the intended behavior, but I would like to keep the current processes running (no restart). Is there a way to disable this behavior?

Python 3.6.2 Anaconda custom (64-bit) Jul 20 2017
Ipython 6.1.0
VS Community 2017 15.4.4
Python 3.0.17310.2

**Comment by huguesv** [comment]
That was the intended behavior, but I agree that this isn't flexible enough. We made the opposite decision when we implemented the equivalent in R (user has to reset explicitely).

This is similar to what Karthik and I intended when we filed https://github.com/Microsoft/PTVS/issues/3347 but that lacks some details so I'll keep both open for now.

Opening the file, CTR+A, CTRL+ENTER is the fastest way to do it AFAIK.
